src/consumer/consumer.js

```diff
@@ -616,9 +616,27 @@ export default class Consumer {
       ? [...writtenComponents, ...R.flatten(writtenDependencies)]
       : writtenComponents;
     this.linkComponents(allComponents, bitMap);
+    await this.reLinkDirectlyImportedDependencies(writtenComponents, bitMap);
     return allComponents;
   }
 
+  /**
+   * an IMPORTED component might be NESTED before.
+   * find those components and re-link all their dependents
+   */
```

Comment:
add the args / return to doc

Reply:
It's a decision we need to make.
We use Flow to document the args and return type. When there is nothing to document about the params except their type, re-documenting the type in the comment, feels to me as a duplication.
Every change in the args or return type needs to be done twice, in Flow and in the docs.